### ISSUE TITLE
room.onPeerStream not firing

### ISSUE THREAD

**Issue by Dletta** [dmotz/trystero#3]
Hi @dmotz ,

When I room.addStream on one peer, I do not see the onPeerStream firing.
```javascript
room = joinRoom(config, ns);
[sendAudio, getAudio] = room.makeAction("audio");
room.onPeerStream((stream, id) => console.log(`got stream from ${id}`, stream));
```
That's how I set the room up. room.onPeerStream doesn't fire for me.

Click function that is supposed to create stream below
```javascript
      talking = await navigator.mediaDevices.getUserMedia({audio: true, video: false});
      console.log('ship out',talking)
      room.addStream(talking,selfId);
```

Add stream is called, in one peer, but the other does not see the stream or add the stream

**Comment by dmotz** [comment]
I just released 0.7.9 on npm which addresses a bug with passing streams. Could you update and try with that?

Otherwise, in your second example you're calling `addStream()` with `selfId` as the second argument which won't work as you can't connect to yourself within the same page -- connecting between multiple tabs will work fine though. Either don't pass a second argument when calling `addStream()` (which sends the stream to everyone) or pass the ID you get from `onPeerJoin()`.

**Comment by Dletta** [comment]
Hi Dan,

I updated to 0.7.9 and now the onPeerStream is firing. I did change the selfId right after I put in issue and since it had persisted I did not update.
